addOns/reports/src/main/zapHomeFiles/reports/high-level-report/resources/main.css

```diff
@@ -0,0 +1,134 @@
+/*body {
```

Comment:
Drop the part that's bock commented?

Reply:
Sure thing. Sorry for leaving those there

Reply:
Making another commit, with the correct .css file. A regrettable oversight